app/models.py

```diff
@@ -1116,6 +1116,7 @@ class Mailbox(db.Model, ModelMixin):
     user_id = db.Column(db.ForeignKey(User.id, ondelete="cascade"), nullable=False)
     email = db.Column(db.String(256), unique=True, nullable=False)
     verified = db.Column(db.Boolean, default=False, nullable=False)
+    force_spf = db.Column(db.Boolean, default=True, nullable=False)
```

Comment:
I think we can add the `server_default` option here so alembic can auto-generate the SQL migration correctly.